### DISCUSSION TITLE
Limiting Backup Storage Size

### DISCUSSION THREAD

**Discussion by XZtotal** [pelican-dev/panel#472] [Ideas]
## Limiting Backup Storage Size
We propose an enhancement for the Pelican panel to include a feature that allows administrators **to limit the total storage space** occupied by **all backups of a server**. This feature would work similarly to the existing backup count limit but would focus on the **cumulative size** of the backups.

**Comment by Alekceii** [comment]
Nice Idea!

**Comment by rmartinoscar** [comment]
Nice to have but not priority atm
https://github.com/pterodactyl/panel/issues/3996#issuecomment-1134786546
